tests/optimagic/optimization/test_nevegrad.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+from numpy.testing import assert_array_almost_equal as aaae
+
+import optimagic as om
+
+
+def test_x0_works_in_minimize():
+    res = om.minimize(
+        fun=lambda x: x @ x,
+        x0=np.arange(3),
+        algorithm="nevergrad_oneplusone",
+    )
+    aaae(res.params, np.zeros(3))
+
+
+def test_x0_works_in_maximize():
+    res = om.maximize(
+        fun=lambda x: -x @ x,
+        x0=np.arange(3),
+        algorithm="nevergrad_oneplusone",
+    )
+    aaae(res.params, np.zeros(3))
```

Comment:
Both of these tests are essentially duplicates of tests we already run for all optimizers (e.g. [here](https://github.com/optimagic-dev/optimagic/blob/main/tests/optimagic/optimization/test_many_algorithms.py). For algorithm specific test it would be better to write tests that show that the algorithm specific parameters have their intended effects. See [here](https://github.com/optimagic-dev/optimagic/blob/main/tests/optimagic/optimizers/test_tao_optimizers.py) for examples. 

If you don't find ways to test algorithm specific behavior, you can simply delete the test file and rely on our standard tests.